C7GameData/RulesFormat.cs

```diff
@@ -5,11 +5,55 @@ namespace C7GameData
     (e.g. in a mod) or as part of the save file. At least for now.
 */
 {
-    using System;
-    public class C7RulesFormat {
-        public string Version { get; set; }
-        public C7RulesFormat() {
-            Version = "v0.0early-prototype";
-        }
-    }
+	using System.Collections.Generic;
+	using System.Text.Json.Serialization;
+
+	public struct StrengthBonus
+	{
+		public string description;
+		public double amount; // e.g. 0.25 = +25% strength
+
+		// Converts a list of strength bonuses to a multiplier on strength. For example, a list of two bonuses of +10% and +25% would return a
+		// multiplier of 1.35. Multipliers cannot be less than zero.
+		public static double ListToMultiplier(IEnumerable<StrengthBonus> bonuses)
+		{
+			double m = 1.0;
+			foreach (StrengthBonus bonus in bonuses)
+				m += bonus.amount;
+			return (m >= 0.0) ? m : 0.0;
+		}
+	}
+
+	public class C7RulesFormat
+	{
+		public string Version { get; set; }
+
+		public List<ExperienceLevel> experienceLevels = new List<ExperienceLevel>();
+		public string defaultExperienceLevelKey;
+		[JsonIgnore]
+		public ExperienceLevel defaultExperienceLevel;
+
+		// TODO: Make sure these numbers are correct
```

Comment:
in fact, make 'em configurable per level

Reply:
Sure, actually we have no choice. I looked around for the exact promotion chances and found this post from someone at Firaxis:
https://forums.civfanatics.com/threads/promotion-probabilities.26048/#post-352717
In the original game promotion chance varies by experience level.